esphome/components/wifi_info/text_sensor.py

```diff
@@ -15,6 +16,9 @@
 IPAddressWiFiInfo = wifi_info_ns.class_(
     "IPAddressWiFiInfo", text_sensor.TextSensor, cg.Component
 )
+ScanResultsWiFiInfo = wifi_info_ns.class_(
+    "ScanResultsWiFiInfo", text_sensor.TextSensor, cg.Component
```

Comment:
```suggestion
    "ScanResultsWiFiInfo", text_sensor.TextSensor, cg.PollingComponent
```

For the updated base class.

Reply:
Both suggestions applied, thanks.